modules/Components/src/client/ACASThingBrowser.coffee

```diff
@@ -163,15 +189,34 @@ class ACASThingBrowserCellController extends Backbone.View
 	render: =>
 		$(@el).empty()
 
-		value = @model.get(@configs.key)
-		if value instanceof Value
-			content = value.escape("value")
-			if value.get("lsType") == "dateValue"  && !@configs.formatter?
-				content = UtilityFunctions::convertMSToYMDDate(content)
-		else if value instanceof Label
-			content = value.escape("labelText")
+		if @model instanceof Backbone.Model
+
+			# The model could be a thing or just a backbone model with key value pairs
+			value = @model.get(@configs.key)
+			# Render Thing Value
+			if value instanceof Value
+				content = value.get("value")
+				# If it's a string then escape it
+				if typeof(content) == "string"
+					content = value.escape(content)
+			# Render Thing Label
+			else if value instanceof Label
+				content = value.escape("labelText")
+			# Render key value backbone model
+			else 
+				# This is not a Thing Value or Label so assume it's just a backbone model with key/value pairs
+				# If the thing model class was passed in then we can use it to determine if this is a date value we should parse
+				content = @model.get(@configs.key)
+				if typeof(content) == "string"
+					content = @model.escape(@configs.key)
 		else
-			content =  @model.escape(@configs.key)
+			# This is strictly a key/pair object
+			content = @model[@configs.key]
+
+		if @options.thingModelExample?
+			modelValue = this.options.thingModelExample.get(@configs.key)
+			if modelValue instanceof Value && modelValue.get("lsType") == "dateValue" && !@configs.formatter?
```

Comment:
This particular line is important for date values.  We now pass one default plain instantiated ls thing to this method so that we can look up the keys and determine if we should format the returned values as dates.